tracer/test/Datadog.Trace.Security.Unit.Tests/EncoderUnitTests.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace Datadog.Trace.Security.Unit.Tests;
 
+[Collection("SecuritySecuentialTests")]
```

Comment:
Did you create a `CollectionDefinition` somewhere? I can't seem to find it otherwise, this doesn't change anything to parallelism

Reply:
Also I dont think it's worth adding sequential runs, it would slow down CI and we so far could not prove there was anything related with parallelism, apart from the 1.10 bug which happens even sequentially, they've always passed without flakiness..

Reply:
Oh, the `CollectionDefinition` didn't uploaded for whatever reason. The main point in making the tests in series was ensuring the possible paralellization didn't affect the memory measuring tests. We haven't had issues until now, but maybe this version is more resource hungry, and it seems son VMs go low on resources.

Reply:
makes sense, also NIT `SecuritySecuentialTests > SecuritySequentialTests` 😅